subscriptions.go

```diff
@@ -191,6 +192,9 @@ type UpdateSubscription struct {
 	NetTerms               NullInt              `xml:"net_terms,omitempty"`
 	PONumber               string               `xml:"po_number,omitempty"`
 	SubscriptionAddOns     *[]SubscriptionAddOn `xml:"subscription_add_ons>subscription_add_on,omitempty"`
+	GatewayCode            string               `xml:"gateway_code,omitempty"`
+	CustomerNotes          string               `xml:"customer_notes,omitempty"`
+	CustomFields           *CustomFields        `xml:"custom_fields,omitempty"`
```

Comment:
Similar question here. I see this available in the "Update Subscription Notes" call. Should this instead be added to the `SubscriptionNotes` type?

Reply:
see above,  we should put it in here directly.